apps/web/src/server/app/app.router.js

```diff
@@ -1,54 +1,35 @@
 import config from '@pins/web/environment/config.js';
 import { Router as createRouter } from 'express';
-import { viewHomepage, viewUnauthenticatedError } from './app.controller.js';
-import { assertGroupAccess, assertIsAuthenticated } from './auth/auth.guards.js';
+import { installAuthMock } from '../../../testing/mocks/auth.js';
+import appealsRouter from '../appeals/appeals.router.js';
+import lpaRouter from '../appeals/case-officer/case-officer.router.js';
+import { handleHeathCheck, viewHomepage, viewUnauthenticatedError } from './app.controller.js';
+import { assertIsAuthenticated } from './auth/auth.guards.js';
 import authRouter from './auth/auth.router.js';
-import { registerCaseOfficerLocals } from './case-officer/case-officer.pipes.js';
-import lpaRouter from './case-officer/case-officer.router.js';
-import { registerInspectorLocals } from './inspector/inspector.pipes.js';
-import inspectorRouter from './inspector/inspector.router.js';
-import { registerValidationLocals } from './validation/validation.pipes.js';
-import validationRouter from './validation/validation.router.js';
 
 const router = createRouter();
 
+// In development only, integrate with locally defined user groups
+
+if (config.authDisabled) {
```

Comment:
I reckon the check for what Auth to use should be done somewhere else - maybe in `assertIsAuthenticated`